src/main/java/kr/sesac/aoao/server/todo/repository/TodoFolderEntity.java

```diff
@@ -48,4 +50,16 @@ public TodoFolderEntity(String content, LocalDate date, UserEntity user, Palette
         this.user = user;
         this.palette = palette;
     }
+
+    public void update(UserEntity user, String content, PaletteEntity palette) {
+        validateUserIsWriter(user);
+        this.content = content;
+        this.palette = palette;
+    }
```

Comment:
```suggestion
    public void update(UserEntity user, String content, PaletteEntity palette) {
        validateUserIsWriter(user);
        this.content = content;
        this.palette = palette;
    }
```

userEntity에서 작성자를 검증을 service 쪽이 아닌 entity에서 작성한 이유가 뭔가용?

Reply:
만약 서비스쪽에서 한다면 getter 를 호출해서 하게되므로 이는 객체지향적이지 않다고 생각했고, 서비스도 비대해질 것이라 생각해서 도메인 안에서 해결했습니당!!